src/scan/familyCheckinSummary/icons/PointsCollectedIcon.tsx

```diff
@@ -0,0 +1,15 @@
+import * as Styled from './style';
+
+type TProps = {
+  count: number;
+};
+
+export const PointsCollectedIcon = ({ count }: TProps) => {
```

Comment:
could you use the same as on ScanSucces?

Reply:
The icon in `ScanSuccess` seems to be much bigger than the one we need here.